DNN Platform/Library/Entities/Users/Profile/ProfilePropertyAccess.cs

```diff
@@ -49,7 +58,7 @@ public CacheLevel Cacheability
         /// <param name="accessingUser">The accessing user.</param>
         /// <param name="targetUser">The target user.</param>
         /// <returns><c>true</c> if property accessible, otherwise <c>false</c>.</returns>
-        public static bool CheckAccessLevel(PortalSettings portalSettings, ProfilePropertyDefinition property, UserInfo accessingUser, UserInfo targetUser)
+        public static bool CheckAccessLevel(IPortalSettings portalSettings, ProfilePropertyDefinition property, UserInfo accessingUser, UserInfo targetUser)

```

Comment:
We need to keep and deprecate the `PortalSettings` overload

Reply:
Makes total sense, implemented that suggestion.